File: backend/candidate-profile-service/src/main/java/com/jobportal/candidateprofile/dto/ai/ExtractedResumeDto.java
package com.jobportal.candidateprofile.dto.ai;

import lombok.*;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class ExtractedResumeDto {
    private String summary;
    private List<String> skills;
    private List<ExtractedExperienceDto> experiences;
    private List<ExtractedEducationDto> educations;
    private List<ExtractedProjectDto> projects;
}

File: backend-dotnet/Services/JobPortal.CandidateService/DTOs/AI/ExtractedResumeDto.cs
using System.Collections.Generic;

namespace JobPortal.CandidateService.DTOs.AI;

public class ExtractedResumeDto
{
    public string? Summary { get; set; }
    public List<string> Skills { get; set; } = new();
    public List<ExtractedExperienceDto> Experiences { get; set; } = new();
    public List<ExtractedEducationDto> Educations { get; set; } = new();
    public List<ExtractedProjectDto> Projects { get; set; } = new();
}
